backend/utils/route_helpers.py

```
@@ -0,0 +1,126 @@
+    Args:
+        e: The exception that occurred
+        operation_name: Name of the operation that failed
+        user_id: Optional user ID for logging
+        **extra_fields: Additional fields to include in logging
+        
+    Returns:
+        tuple: (error_response, error_code)
+    """
+    try:
+        jwt_user = get_jwt_identity()
+    except Exception:  # safe fallback if no request/JWT context
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)